lcls2_producers/prod_config_ued.py

```diff
@@ -60,8 +60,8 @@ def get_droplet2photon(run):
 # epicsPV = ['las_fs14_controller_time']
 # epicsOncePV = ['m0c0_vset', ('TMO:PRO2:MPOD:01:M2:C3:VoltageMeasure', 'MyAlias'),
 #               'IM4K4:PPM:SPM:VOLT_RBV', "FOO:BAR:BAZ", ("X:Y:Z", "MCBTest"), "A:B:C"]
-epicsPV = []
-epicsOncePV = []
+epicsPV = ["UED:USR:TCT:02:GET_TEMP_A"]
```

Comment:
You did not define an epicsArchPVs variable here. THe PV in that list are to be added from the epics archiver, which is not what they want here I believe.